run_flower.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+su -m myuser -c "celery flower --broker=pyamqp://admin:rabbitmq@rabbit:5762//"
```

Comment:
does this need to change to `guest:guest`? How will this work when we run on production, I wonder? Will it have to grab user/pass from env vars or something?